### ISSUE TITLE
Package for Fedora

### ISSUE THREAD

**Issue by edigiacomo** [ARPA-SIMC/PRAGA#8]
PRAGA is currently packaged only for CentOS 8. The Fedora (>=34) package requires some investigation in the `BuildRequires` because `qt5-devel` was dropped since Fedora 32.

**Comment by edigiacomo** [comment]
It seems that PRAGA requires only `qt5-qtbase-devel` and `qt5-qtcharts-devel`

```
$ git grep -h "include <Q" | \
sed -e 's,^ *# *include,,g' -e 's,/.*,,g' | \
sort -u | \
while read i ; do rpm -qf $(find /usr/include/qt5 -name "$i*" | head -n1); done | \
sort -u

qt5-qtbase-devel-5.14.2-6.fc32.x86_64
qt5-qtcharts-devel-5.14.2-1.fc32.x86_64
```

**Comment by edigiacomo** [comment]
Fixed in 1b8c613c7439723568edda1796a09ff139440091
